aspnet/client-side/bower/samples/Starter/.bowerrc

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
I mean the entire sample. If you don't have any literal-includes from the sample code then I don't see any reason to have it in the repo.